test/integration-orientdb/tests/define/indexes.js

```diff
@@ -39,5 +40,18 @@ describe('Define related Operations', function() {
       });
     });
 
+    it('should properly create Link property from collection', function(done) {
```

Comment:
How about 'should properly create Link property in one-to-many association', just so we know/remember the specific objective of this test.